tests/integration/cmor/_fixes/cmip6/test_cesm2_waccm.py

```diff
@@ -124,3 +125,74 @@ def test_get_tas_fix():
 def test_tas_fix():
     """Test fix for ``tas``."""
     assert Tas is BaseTas
+
+
+class TestTasmin(unittest.TestCase):
+    """Test tasmin fixes."""
+
+    def setUp(self):
+        """Prepare tests."""
+        wrong_time_coord = iris.coords.AuxCoord(
+            points=[1.0, 2.0, 1.0, 2.0, 3.0],
+            var_name="time",
+            standard_name="time",
+            units="days since 1850-01-01",
+        )
+
+        correct_time_coord = iris.coords.AuxCoord(
+            points=[1.0, 2.0, 3.0],
+            var_name="time",
+            standard_name="time",
+            units="days since 1850-01-01",
+        )
+
+        lat_coord = iris.coords.DimCoord(
+            [0.0],
+            standard_name="latitude",
+            var_name="lat",
+        )
+
+        lon_coord = iris.coords.DimCoord(
+            [0.0],
+            standard_name="longitude",
+            var_name="lon",
+        )
+
+        self.time_coord = correct_time_coord
+        self.wrong_cube = iris.cube.CubeList(
+            [iris.cube.Cube(np.ones((5, 1, 1)), var_name="tasmin", units="K")]
+        )
+        self.wrong_cube[0].add_aux_coord(wrong_time_coord, 0)
+        self.wrong_cube[0].add_dim_coord(lat_coord, 1)
+        self.wrong_cube[0].add_dim_coord(lon_coord, 2)
+        self.correct_cube = iris.cube.CubeList(
+            [iris.cube.Cube(np.ones(3), var_name="tasmin", units="K")]
+        )
+        self.correct_cube[0].add_aux_coord(correct_time_coord, 0)
+
+        self.fix = Tasmin(None)
+
+    def test_get(self):
+        """Test fix get."""
+        self.assertListEqual(
+            Fix.get_fixes("CMIP6", "NCAR", "day", "tasmin"),
```

Comment:
hi @Karen-A-Garcia - these tests have not been corrected: they are still in UnitTest format, but a bit more critical, are not testing for the correct model, and also have issues with the actual tested outcome vs expected, could you please have another look at them? My code suggestion change appears to have been resolved, but the model is still wrong: you should test for CESM2-WACCM, not NCAR. Perhaps you forgot to push the changes? :beer:

Reply:
Oh that is so strange because it was changed on my files when I pushed them and they seemed to not have change for some reason. I'll try to fix that.

Reply:
No worries at all! I was suspecting it's a push issue. If it's too much trouble to convert the tests to simple pytest funcs, don't worry about it, it's great you plopped tests in anyways. Let me know if I can help in any way 👍🍺